stl/inc/barrier

```diff
@@ -93,9 +93,8 @@ public:
     }
 
     _NODISCARD_BARRIER_TOKEN arrival_token arrive(ptrdiff_t _Update = 1) noexcept /* strengthened */ {
-        // Shifting before precondition check, so that exceeding max() will trigger precondition check too
+        _STL_VERIFY(_Update > 0 && _Update <= (max) (), "Precondition: update > 0 (N4950 [thread.barrier.class]/12)");

```

Comment:
Sorry for this late review -- I didn't realize it was "Pending"-ed by github.

`_Update <= max()` it necessary, as it avoids large positive value becoming negative after shift. However, this will cause a mismatch between the check and error message. It is more relevant to `"update<= expected count"`, but I don't think we should put off this check. I have no idea how to make an improvement. Any ideas?

Reply:
I'm okay with a slightly confusing message here.